site/sfguides/src/data_engineering_deploying_pipelines_with_snowflake_and_dbt_labs/data_engineering_deploying_pipelines_with_snowflake_and_dbt_labs.md

```diff
@@ -0,0 +1,369 @@
+authors: Dmytro Yaroshenko
+id: data_engineering_deploying_pipelines_with_snowflake_and_dbt_labs
+summary: This guide will provide step-by-step details for building and deploying data engineering pipelines with Snowflake and dbt labs
+categories: featured,data-engineering,notebooks
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Data Engineering, Notebooks, dbt
+
+# Deploying pipelines with Snowflake and dbt labs
```

Comment:
@sfc-gh-dyaroshenko Please change the title to "Deploying Pipelines with Snowflake and dbt Labs"